server/services/reporting/util.ts

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable camelcase */
+import dayjs from 'dayjs';
+import { addYearToDate } from '../../utils';
+
+interface RosEntry {
+  participant_id;
+  participantJoin: {
+    body: {
+      firstName: string;
+      lastName: string;
+    };
+  }[];
+  siteJoin: {
+    body: {
+      siteName: string;
+      healthAuthority: string;
+    };
+  };
+  data: {
+    date: Date | string;
+    startDate: Date | string;
+    positionType?: string;
+    employmentType?: string;
+  };
+}
+
+/**
+ * Validation layer to see if the user has access to requested health region
+ * @param user data of a user who accesses the endpoints
+ * @param regionId health region
+ * @return true if the user has access
+ */
+export const checkUserRegion = (user, regionId: string): boolean =>
+  user && user.regions?.includes(regionId);
+
+export const mapRosEntries = (rosEntries: RosEntry[]) =>
```

Comment:
This (and the interface) feels out of place in a utils file. Feeling it more in a ROS specific file or your existing milestones one